core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/CodeSystemCreateRequest.java

```diff
@@ -112,25 +120,25 @@ private void checkBranchPath(final RepositoryContext context, final boolean crea
 		// codeSystemId checked against all resources
 		final boolean existingId = ResourceRequests.prepareSearch()
 			.setLimit(0)
-			.filterById(id)
+			.filterById(getId())
 			.build()
 			.execute(context)
 			.getTotal() > 0;
 			
 		if (existingId) {
-			throw new AlreadyExistsException("Resource", id);
+			throw new AlreadyExistsException("Resource", getId());
 		}
 		
 		// title should be unique across all resources
 		final boolean existingTitle = ResourceRequests.prepareSearch()
 			.setLimit(0)
-			.filterByTitleExact(title)
+			.filterByExactTermIgnoreCase(getTitle())
```

Comment:
It should be filterByTitleExactIgnoreCase not `term`. There is no such field in the Resource model.